e2e/renderer/e2e/ngfor.e2e-spec.ts

```diff
@@ -2,45 +2,39 @@ import {
     AppiumDriver,
     createDriver,
     SearchOptions,
+    UIElement
 } from "nativescript-dev-appium";
 
 import { isAbove } from "./helpers/location";
-import { DriverWrapper, ExtendedUIElement } from "./helpers/appium-elements";
 
 describe("ngFor scenario", () => {
     let driver: AppiumDriver;
-    let driverWrapper: DriverWrapper;
-    let addButton: ExtendedUIElement;
-    let removeButton: ExtendedUIElement;
-    let elements: ExtendedUIElement[] = [];
+    let addButton: UIElement;
+    let removeButton: UIElement;
+    let elements: UIElement[] = [];
     let lastAddedElementId = 0;
 
     before(async () => {
         driver = await createDriver();
-        driverWrapper = new DriverWrapper(driver);
-    });
-
-    after(async () => {
-        await driver.quit();
-        console.log("Driver quits!");
+        await driver.driver.resetApp();
```

Comment:
Can we add `resetApp` method to the `AppiumDriver` class that internally invokes `_driver.resetApp()`? (this is a change in the appium plugin)